src/hotspot/os/windows/os_windows.cpp

```diff
@@ -6204,3 +6204,115 @@ const void* os::get_saved_assert_context(const void** sigInfo) {
   *sigInfo = nullptr;
   return nullptr;
 }
+
+#ifdef HAVE_PLATFORM_PRINT_NATIVE_STACK
```

Comment:
Saint, this will include the following code even if HAVE_PLATFORM_PRINT_NATIVE_STACK is defined as 0, I suggest either
changing it to #if HAVE_PLATFORM_PRINT_NATIVE_STACK == 1; or keep #ifdef and change the definition to #define HAVE_PLATFORM_PRINT_NATIVE_STACK 
^no value